lib/fs.js

```diff
@@ -17,6 +18,22 @@ const mkdir = (dir, mode, cb) => {
   });
 };
 
+const recursivlyListDirs = dir => {
+  const list = [dir];
+  let nextDir = dir;
+  const root = dir.startsWith('/') ? '/' : '.';
+  while ((nextDir = path.dirname(nextDir)) !== root) list.push(nextDir);
+  return list;
+};
+
+const rmdirp = (dir, cb) => {
+  metasync.series(
+    recursivlyListDirs(dir),
+    (dirName, cb) => fs.rmdir(dirName, cb),
```

Comment:
I don't think we should return the value obtained from calling the `fs.rmdir()` here:
```suggestion
    (dirName, cb) => {
      fs.rmdir(dirName, cb);
    },
```

Reply:
What about just `fs.rmdir`?
```js
    fs.rmdir,
```